src/main/java/dataStructures/hashSet/openAddressing/HashSet.java

```diff
@@ -99,19 +103,28 @@ public boolean add(T element) {
      */
     public boolean remove(T element) {
```

Comment:
double check, this ignore TombStone encountered right

Reply:
Yup. `remove` ignores Tombstones encountered, since in the probe sequence {1, 2, ...  m}, there is a possibility that the element to be removed is in bucket m, with multiple elements in between that are removed.

Reply:
nice